Dockerfile

```diff
@@ -55,8 +57,11 @@ RUN        touch node_modules
 ARG        commit_sha=(unknown)
 RUN        true \
            && CALYPSO_ENV=production COMMIT_SHA=$commit_sha npm run build \
-           && chown -R nobody /calypso \
+           && find . -not -path './node_modules/*' -print0 | xargs -0 chown nobody \
            && true
 
+# Print files, just for testing
+RUN        find . -not -user nobody | wc -l
+
```

Comment:
This needs to be removed, but is helpful to verify that `chown` has worked correctly.